about.html

```diff
@@ -32,32 +32,36 @@ <h1 class="section-header">About Speedometer 3</h1>
                                 <li>Measures the time to add, complete, and remove 100 todo items in a basic list.</li>
                                 <li>Each example implements the same todo application (TodoMVC) using different techniques and frameworks.</li>
                                 <li>
-                                    Workloads: <a href="resources/todomvc/vanilla-examples/javascript-es5/readme.md" target="_blank">TodoMVC-JavaScript-ES5</a>,
-                                    <a href="resources/todomvc/vanilla-examples/javascript-web-components/readme.md" target="_blank">TodoMVC-WebComponents</a>,
-                                    <a href="resources/todomvc/architecture-examples/react-complex/readme.md" target="_blank">TodoMVC-React-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/backbone/readme.md" target="_blank">TodoMVC-Backbone</a>, <a href="resources/todomvc/architecture-examples/angular/readme.md" target="_blank">TodoMVC-Angular</a>,
-                                    <a href="resources/todomvc/architecture-examples/vue/readme.md" target="_blank">TodoMVC-Vue</a>, <a href="resources/todomvc/architecture-examples/jquery/readme.md" target="_blank">TodoMVC-jQuery</a>,
-                                    <a href="resources/todomvc/architecture-examples/preact/readme.md" target="_blank">TodoMVC-Preact</a>, <a href="resources/todomvc/architecture-examples/svelte/readme.md" target="_blank">TodoMVC-Svelte</a>,
-                                    <a href="resources/todomvc/architecture-examples/lit/readme.md" target="_blank">TodoMVC-Lit</a>
+                                    Workloads: <a href="resources/todomvc/vanilla-examples/javascript-es5/README.md" target="_blank">TodoMVC-JavaScript-ES5</a>,
```

Comment:
What do y'all think of targeting the README file as hosted on github, so that it can be seen rendered?

Here is how it looks like on browserbench: https://browserbench.org/Speedometer3.0-preview/resources/todomvc/architecture-examples/preact/readme.md

Reply:
Yeah, linking to Github makes sense to me.